appveyor.yml

```diff
@@ -3,6 +3,7 @@ image: Visual Studio 2017
 clone_folder: c:\gopath\src\github.com\hashicorp\nomad
 environment:
   GOPATH: c:\gopath
+  GOBIN: c:\gopath\bin
```

Comment:
Oh my, this is necessary for some reason on Windows? I didn't even know GOBIN was a thing.